DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Centre/Configuration/RegistrationPromptsControllerTests.cs

```diff
@@ -386,7 +446,12 @@ public void AddRegistrationPromptSummary_calls_registration_prompt_service_and_r
                         "Test\r\nAnswer"
                     )
                 ).MustHaveHappened();
-                registrationPromptsController.TempData.Peek<AddRegistrationPromptData>().Should().BeNull();
+                A.CallTo(

```

Comment:
Please rename this test and the one below it to `AddRegistrationPromptSummaryPost` instead of `AddRegistrationPromptSummary`